libpgmodeler_ui/src/modeloverviewwidget.cpp

```diff
@@ -26,7 +26,7 @@ ModelOverviewWidget::ModelOverviewWidget(QWidget *parent) : QWidget(parent, Qt::
 	zoom_factor=1;
 	curr_resize_factor=RESIZE_FACTOR;
 	this->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Fixed);
-    setWindowFlags(Qt::Tool);
+	setWindowFlags(Qt::Tool);
```

Comment:
This is a matter of taste but I would add this flag to the constructor's signature since this flag will be a permanent behaviour of the dialog. I mean this:
```cpp
ModelOverviewWidget::ModelOverviewWidget(QWidget *parent) : QWidget(parent, Qt::WindowCloseButtonHint | Qt::Tool)
{
...
```